src/bp-core/bp-core-update.php

```diff
@@ -1387,3 +1390,279 @@ function bb_update_to_1_8_1() {
 		bb_email_queue()::create_db_table();
 	}
 }
+
+/**
+ * update routine.
+ * Migrate default cover images from theme option.
+ *
+ * @since BuddyBoss [BBVERSION]
+ */
+function bb_update_to_1_8_6() {
+	global $buddyboss_theme_options;
+
```

Comment:
Please add those code snippets here:
```
// Do not ignore deprecated code for existing installs.
bp_add_option( '_bp_ignore_deprecated_code', false );
```

it will not skip the deprecated file load for existing installs.